ptsprojects/mynewt/mesh.py

```diff
@@ -42,8 +42,8 @@
 from time import sleep
 
 
-device_uuid = hexlify(uuid4().bytes)
-device_uuid2 = hexlify(uuid4().bytes)
+device_uuid = "00000000000000000000000000000000"
```

Comment:
Why is this change here?

Reply:
Is there a reason that for Mynewt these are random?